test/verify/check-apps

```diff
@@ -94,7 +94,8 @@ class TestApps(PackageCase):
                 '{ "config": { "appstream_data_packages": [ "appstream-data-test" ] } }')
 
         if urlroot != "":
-            m.execute(f'mkdir -p /etc/cockpit/ && echo "[WebService]\nUrlRoot={urlroot}" > /etc/cockpit/cockpit.conf')
+            m.execute("mkdir -p /etc/cockpit")
+            m.write("/etc/cockpit/cockpit.conf", f"[WebService]\nUrlRoot={urlroot}")
```

Comment:
`m.write()` should imply `mkdir -p` of the parent directory.

Reply:
I thought about that, I can't quite make up my mind between "too much magic" and making things simpler. But as you suggest it as well, I'll send a bots PR to imply mkdir, and then do another pass over tests to clean up redundant mkdirs.

Reply:
See https://github.com/cockpit-project/bots/pull/3806 .